__tests__/app.test.js

```diff
@@ -215,5 +215,95 @@ describe("API Testing", () => {
           });
       });
     });
+    describe("PATCH requests", () => {
+      test("Should return a status code of 201 if the update is sucessfull", () => {
+        const reqBody = {
+          inc_votes: 10,
+        };
+        return request(app).patch("/api/reviews/1").send(reqBody).expect(201);
+      });
+      test("Should return the review with the updated vote increment", () => {
+        const reqBody = {
+          inc_votes: 1,
+        };
+        return request(app)
+          .patch("/api/reviews/4")
+          .send(reqBody)
+          .expect(201)
+          .then((res) => {
+            expect(res.body.review).toHaveLength(1);
```

Comment:
I might be tempted to have this respond with a { review : { .. .review} } rather than an array as we probably only ever want to send one review back and not multiple reviews .... I have said the word review too many reviewing times :)